miniapps/nurbs/makefile

```diff
@@ -109,5 +116,6 @@ clean-build:
 	rm -rf *.dSYM *.TVD.*breakpoints
 
 clean-exec:
-	@rm -f refined.mesh mesh.* sol.* mode_*
+	@rm -f refined.mesh sin-fit.mesh mesh.* sol.* mode_*
 	@rm -rf Example1*
+	@rm -rf CurveInt
```

Comment:
`CurveInt` again

Reply:
Done!

Reply:
The `CurveInt` directory is created for VisIt output, so this `rm -rf CurveInt` should be kept.

Reply:
I was already a bit surprised by it. It is reverted!